opentech/apply/funds/templates/funds/tables/table.html

```diff
@@ -1,29 +1,73 @@
 {% extends 'django_tables2/table.html' %}
-{% load table_tags %}
+{% load django_tables2 table_tags review_tags %}
 
 {# example of how to extend the table template #}
 {% block table.tbody.row %}
-    {{ block.super }}
+    <tr {{ row.attrs.as_html }} data-record-id="{{ row.record.id }}">
+        {% for column, cell in row.items %}
+            <td {{ column.attrs.td.as_html }}>{% if column.localize == None %}{{ cell }}{% else %}{% if column.localize %}{{ cell|localize }}{% else %}{{ cell|unlocalize }}{% endif %}{% endif %}</td>
+        {% endfor %}
+    </tr>
 
     {% with submission=row.record %}
         {# placeholder for example data row #}
-        <tr class="child">
-            <td>Applicant: {{ submission.full_name }}</td>
+        <tr class="child" data-parent-id="{{ row.record.id }}">
+            <td colspan="{{ table.columns|length }}">
+                <table>
+                    <tr>
+                        <th>Applicant</th>
+                        <th>Last updated</th>
+                        <th>Next deadline</th>
+                        <th>Reviewers / Outcomes</th>
+                    </tr>
+                    <tr>
+                        <td>{{ submission.full_name }}</td>
+                        <td>
+                            {% if row.cells.last_update != '—' %}
+                                <strong>by {{ row.cells.last_update.user }}</strong><br/>
+                                {{ row.cells.last_update.timestamp|date:"m.d.y \a\t H:i" }}
+                            {% else %}
+                                {{ row.cells.last_update }}
+                            {% endif %}
+                        </td>
+                        <td>&mdash;</td>
```

Comment:
Do we want to be adding deadlines when this will not show anything yet and is potentially confusing.